src/index.ts

```diff
@@ -519,6 +612,13 @@ export class UnleashClient extends TinyEmitter {
             }
         }
     }
+
+    private emitReady() {
```

Comment:
@jeremiewtd pas sur que ça soit tj pertinent cette méthode car a priori maintenant (depuis ma maj depuis main) on peut faire un emit ready sans controler "this.readyEventEmitted" (ln 596)

Reply:
c'est peut-être un bug chez eux non ? 😃

Reply:
ben je sais pas trop, ça a été ajouté vers là:
https://github.com/Unleash/unleash-proxy-client-js/commit/de95fe4a1f10e8d6331c643fa9a471470d42a9b8#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80

de ce que je constate :
- par exemple sur le bootstrap avant on pouvait faire une emit(Ready mais sans maj du flag interne
- fetchedFromServer semble remplacer des fois readyEventEmitted dans le code de la lib
- readyEventEmitted ne semble plus servir dans le code de la lib (index), la valeur est accessible uniquement via    public isReady(): boolean { return this.readyEventEmitted;} ...